bundles/org.openhab.binding.valloxmv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -246,4 +258,86 @@
 		<category>Temperature</category>
 		<state pattern="%.2f %unit%" readOnly="false" />
 	</channel-type>
+
+	<channel-type id="boosttime" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Boost profile timer value</label>
```

Comment:
The first letter of every word in a label, with a couple of exceptions, should be capitalized.

Reply:
I'm sorry, I'm not sure what labels are you here referring to?

Reply:
Got that after second look, will shorten them as well.. e.g. in above "Boost Timer" should explain, etc.